internal/testutil/acc/encryption_at_rest.go

```diff
@@ -153,3 +157,65 @@ func EARImportStateIDFunc(resourceName string) resource.ImportStateIdFunc {
 		return rs.Primary.ID, nil
 	}
 }
+
+// EncryptionAtRestExecution creates an encryption at rest configuration for test execution.
+func EncryptionAtRestExecution(tb testing.TB) string {
```

Comment:
can be good to clarify it uses a fixed project and what env variables are leveraged. Potentially passing values as parameter to the function also makes it more explicit.